piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java

```diff
@@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
     }
 
     /**
+     * Configure Piwik to receive more than 5 custom variable
+     * http://piwik.org/faq/how-to/faq_17931/
+     * Changing this limit will affect all Tracker instances
+     * @param maxVariables
+     */
+    public void setMaxCustomVariables(int maxVariables){
+        CustomVariables.MAX_VARIABLES = maxVariables;
```

Comment:
I don't like the use of a static variables like this. I'm on mobile ATM so I can't browse code well, can't we pass this as configuration parameter before tracker init? What do you think?